manifests/services/nulog-inference-service-control-plane.yaml

```diff
@@ -39,4 +40,4 @@ spec:
             cpu: "7.5"
           requests:
             memory: 1Gi
-            cpu: "3.5"
\ No newline at end of file
+            cpu: "1"
```

Comment:
@tybalex is this fine?

Reply:
@galal-hussein can we apply `1 cpu` for Quickstart setting and more cpus for production setting?

Reply:
sure will fix